src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -342,13 +342,15 @@ Computes the water flux given the absolute potential (pressure/(ρg))
 and the conductivity along
 the flow path between these two points.
 
-We currently assuming an arithmetic
-mean for mean K_sat between the two points (Bonan, 2019; Zhu, 2008)
-to take into account the change in K_sat halfway
-between z1 and z2; this is incorrect for compartments of differing sizes.
-"""
-function flux(z1, z2, ψ1, ψ2, K1, K2)
-    flux = -(K1 + K2) / 2 * ((ψ2 - ψ1) / (z2 - z1) + 1)
+We currently assuming an geometric
+mean for mean K_sat between the two points. Previously,
+we used the arithmetic mean (Bonan, 2019; Zhu, 2008),
+but then when the soil K was very low, root extraction would
+continue. This should be modified for compartments of differing sizes.
+"""
+function flux(z1::FT, z2::FT, ψ1::FT, ψ2::FT, K1::FT, K2::FT) where {FT}
```

Comment:
I think a more descriptive name than "flux" would be nice - maybe "water_flux"? up to you

Reply:
+1

Reply:
I changed this